src/Collection/Collection.router.spec.ts

```diff
@@ -1374,6 +1374,78 @@ describe('Collection router', () => {
         })
       })
 
+      describe('and interating with the database fails', () => {
+        let itemCurationDeleteSpy: jest.SpyInstance<
+          ReturnType<typeof ItemCuration.delete>
+        >
```

Comment:
With the change, instead of `ItemCuration.delete`, `ItemCuration.deleteByIds` is being used.